pkg/istio-agent/xds_proxy.go

```diff
@@ -293,6 +310,9 @@ func (p *XdsProxy) HandleUpstream(ctx context.Context, con *ProxyConnection, xds
 	go p.handleUpstreamRequest(ctx, con)
 	go p.handleUpstreamResponse(con)
 
+	// Handle ECDS update asynchronously, it might need to download Wasm modules.
+	go p.handleUpstreamECDSResponse(con)
```

Comment:
nit: we don't need to run this goroutine if `features.WasmRemoteLoadConversion` is `false`

Reply:
Done.